lib/htmllint.js

```diff
@@ -23,18 +30,45 @@ module.exports = function(config, done) {
     return str;
   }
 
+  // copy environment but filter java options to pass on the command line
+  function cleanEnvironment() {
+    Object.keys(process.env)
+      .filter(function(property){
+        if(property === 'JAVA_TOOL_OPTIONS' || property === '_JAVA_OPTIONS'){
```

Comment:
There's a bunch of spacing issues, like the missing spaces after `if` and before the `{`. Can you fix these and the others?